histomicstk/cli/NucleiDetection/NucleiDetection.xml

```diff
@@ -218,4 +218,17 @@
       <default>1</default>
     </integer>
   </parameters>
+    <parameters advanced="true">
+    <label>Color inversion</label>
+    <description>Choose if color inversion is needed</description>
+    <string-enumeration>
```

Comment:
There is a boolean type which would be more appropriate.  But, we might want a 3-state value: unspecified or "default" or something similar to invert on single band and not on RGB, and explicit true/false or yes/no values to override the default.

Reply:
I have updated the color inversion tool box to have one more option as you recommended "unspecified". right now unspecified option points to no color inversion.